src/extension.ts

```diff
@@ -50,8 +57,10 @@ export function activate(context: ExtensionContext) {
     /**
      * Delegate notifications from Java JDT LS to the MicroProfile LS
      */
-    bindNotification('microprofile/propertiesChanged');
-
+    context.subscriptions.push(commands.registerCommand('microprofile/propertiesChanged', (event: any) => {
```

Comment:
IMO, it would be more clear if we did `(event: MicroProfilePropertiesChangeEvent) => {` instead of `(event: any) => {`

Reply:
fixed